src/averageAges.js

```diff
@@ -14,12 +14,25 @@
  *
  * @return {number}
  */
+function getCentury(person) {
+  return Math.ceil(person.died / 100);
+}
+
+function getPeopleBySex(people, sex) {
+  return people.filter(person => person.sex === sex);
+}
+
+function getAverageAge(people) {
+  return people
+    .map(person => person.died - person.born)
+    .reduce((a, b) => a + b, 0) / people.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const fromCentury = people.filter(person => getCentury(person) === century);
+  const scope = century === undefined ? people : fromCentury;
```

Comment:
would be easier that way
```suggestion
  const scope = !century ? people : fromCentury;
```